setup.cfg

```diff
@@ -27,12 +27,12 @@ console_scripts =
    pbench-clear-results = pbench.cli.agent.commands.results.clear:main
    pbench-clear-tools = pbench.cli.agent.commands.tools.clear:main
    pbench-config = pbench.cli.getconf:main
-   pbench-copy-results = pbench.cli.agent.commands.results:main
    pbench-generate-token = pbench.cli.agent.commands.generate_token:main
    pbench-list-tools = pbench.cli.agent.commands.tools.list:main
    pbench-list-triggers = pbench.cli.agent.commands.triggers.list:main
-   pbench-move-results = pbench.cli.agent.commands.results:main
    pbench-register-tool-trigger = pbench.cli.agent.commands.triggers.register:main
+   pbench-results-copy = pbench.cli.agent.commands.results:main
+   pbench-results-move = pbench.cli.agent.commands.results:main
```

Comment:
Do we really want two commands for this function?  If not, what do we want the one command to be?

Reply:
The `copy` version is meant to push the dataset but keep the tarball locally while `move` would delete it and free the local space. Is the implementation not doing that?

```
    if [[ "$script_name" == "pbench-move-results" ]]; then
	rm -r ${result_dir}
	if [[ ${?} -ne 0 ]]; then
	    error_log "UNEXPECTED ERROR: rm failed to remove the ${result_dir} directory hierarchy"
	    exit 1
	fi
    else
	touch ${result_dir}.copied
	if [[ ${?} -ne 0 ]]; then
	    error_log "UNEXPECTED ERROR: touch failed to ${result_dir}.copied"
	    exit 1
	fi
    fi
```

Reply:
The current implementation is removing everything.  (Thanks for the prompt -- I found another bit of the external MD5 stuff that I'd missed....)

Is it desirable to continue with the `copy` functionality?  (I can trundle off and try to implement that....)  Do we want to implement that as a separate command as we have in the past, or do we want to add it as an option on the existing command?  (I believe that the latter would be better/easier, but I don't know what weight to ascribe to cultural compatibility.)

Reply:
As indicated below, we're going to take a different tack which, for the moment at least, renders this issue safely moot.  :-)

Thank you for pointing it out!